proposals/relaxed-simd/Overview.md

```diff
@@ -224,10 +224,30 @@ def min_or_max(a : v128, b : v128, lanes : int, is_min : bool):
       result[i] = IMPLEMENTATION_DEFINED_ONE_OF(a[i], b[i])
     else:
       result[i] = is_min ? min(a, b) : max(a, b)
+  return result
 ```
 
 Where `IMPLEMENTATION_DEFINED_ONE_OF(x, y)` returns either `x` or `y`, depending on the implementation.
 
+### Relaxed Rounding Q-format Multiplication
+
+- `i16x8.q15mulr_s(a: v128, b: v128) -> v128`
+
+Returns the multiplication of 2 fixed-point numbers in Q15 format. If both
+inputs are `INT16_MIN`, the result overflows, and the return value is
+implementation defined (either wrap around, or saturate).
```

Comment:
IMO we should restrict the set of possible values (i.e. implementation-defined as either `INT16_MIN` or `INT16_MAX`).

Reply:
Good point, fixed in https://github.com/WebAssembly/relaxed-simd/pull/62